packages/lib/src/components/Dropin/elements/createElements.ts

```diff
@@ -20,13 +20,10 @@ const createElements = (
     commonProps,
     core: ICore
 ): Promise<UIElement[]> => {
-    const elements = paymentMethods
+    const elements = optionallyFilterUpiSubTxVariants(paymentMethods)
         .map(paymentMethod => {
-            const paymentMethodConfigurationProps = getComponentConfiguration(
-                paymentMethod.type,
-                paymentMethodsConfiguration,
-                paymentMethod.isStoredPaymentMethod
-            );
+            const isStoredPaymentMethod = 'isStoredPaymentMethod' in paymentMethod && paymentMethod.isStoredPaymentMethod;
```

Comment:
Is it needed to perform this check here? Just wondering that since it wasn't here before

Reply:
I got a TS error `TS2339: Property isStoredPaymentMethod does not exist on type PaymentMethod | StoredPaymentMethod
Property isStoredPaymentMethod does not exist on type PaymentMethod` so I added this extra check.